Training/Task/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rltsquare_config" translate="label" sortOrder="200">
+            <label>RLTSquare Email</label>
+        </tab>
+        <section id="email_template" translate="label" type="text" sortOrder="42" showInDefault="1"
+                 showInWebsite="1" showInStore="1">
+            <class>separator-top</class>
+            <label>Email Template</label>
+            <tab>rltsquare_config</tab>
+            <resource>Training_Task::email_template</resource>
+            <group id="general" translate="label" sortOrder="92" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Email Template</label>
+                <field id="rltsquare_email_template" translate="label" type="select" sortOrder="30" showInDefault="1"
```

Comment:
Overlooked logic - we need to define config.xml with default value set to point to name of our custom email template